lib/jwkset.js

```diff
@@ -0,0 +1,118 @@
+var jws = require('jws')
+var request = require('request')
+var jwk2pem = require('pem-jwk').jwk2pem
+var pem2jwk = require('pem-jwk').pem2jwk
```

Comment:
Doesn't look like Node or io.js have destructuring yet (per http://kangax.github.io/compat-table/es6/), but we could always use [babel](https://babeljs.io/docs/learn-es6/#destructuring) (formerly **6to5**).

```js
'use strict';

var {jwk2pem, pem2jwk} = require('pem-jwk');

var hello = (name) => {
  console.log(`hello ${name}`);
};

// `$ ./node_modules/.bin/babel-node index`

hello('rfk'); // hello rfk

console.log(jwk2pem); // [Function: jwk2pem]
console.log(pem2jwk); // [Function: pem2jwk]
```